pulp_rpm/app/tasks/publishing.py

```diff
@@ -482,15 +485,23 @@ def create_repomd_xml(
 
     repomd = cr.Repomd()
 
-    repomdrecords = [
-        ("primary", pri_xml_path, pri_db),
-        ("filelists", fil_xml_path, fil_db),
-        ("other", oth_xml_path, oth_db),
-        ("primary_db", pri_db_path, None),
-        ("filelists_db", fil_db_path, None),
-        ("other_db", oth_db_path, None),
-        ("updateinfo", upd_xml_path, None),
-    ]
+    if publication.sqlite_metadata:
+        repomdrecords = [
+            ("primary", pri_xml_path, pri_db),
+            ("filelists", fil_xml_path, fil_db),
+            ("other", oth_xml_path, oth_db),
+            ("primary_db", pri_db_path, None),
+            ("filelists_db", fil_db_path, None),
+            ("other_db", oth_db_path, None),
+            ("updateinfo", upd_xml_path, None),
```

Comment:
DBs seem to be added to the xml types, while the db types  have no DB. DO I read this correctly? How did it work before? :/